### ISSUE TITLE
Unable to open Blob Container

### ISSUE THREAD

**Issue by saurk** [microsoft/AzureStorageExplorer#4156]
**Storage Explorer Version**: 
**Build Number**: 
**Platform/OS**: 
**Architecture**: 
**Regression From**:  

## Additional Context ##

```
{
  "name": "RestError",
  "message": "request to https://cgasdlsprod.blob.core.windows.net/raw?restype=container failed, reason: getaddrinfo ENOTFOUND cgasdlsprod.blob.core.windows.net",
  "stack": "RestError: request to https://cgasdlsprod.blob.core.windows.net/raw?restype=container failed, reason: getaddrinfo ENOTFOUND cgasdlsprod.blob.core.windows.net\n    at new RestError (C:\\Users\\sauku\\AppData\\Local\\Programs\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\@azure\\core-http\\dist\\index.js:2247:28)\n    at NodeFetchHttpClient.<anonymous> (C:\\Users\\sauku\\AppData\\Local\\Programs\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\@azure\\core-http\\dist\\index.js:2424:35)\n    at step (C:\\Users\\sauku\\AppData\\Local\\Programs\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\@azure\\core-http\\node_modules\\tslib\\tslib.js:140:27)\n    at Object.throw (C:\\Users\\sauku\\AppData\\Local\\Programs\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\@azure\\core-http\\node_modules\\tslib\\tslib.js:121:57)\n    at rejected (C:\\Users\\sauku\\AppData\\Local\\Programs\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\@azure\\core-http\\node_modules\\tslib\\tslib.js:112:69)\n    at processTicksAndRejections (internal/process/task_queues.js:93:5)",
  "code": "REQUEST_SEND_ERROR",
  "request": {
    "streamResponseBody": false,
    "url": "https://cgasdlsprod.blob.core.windows.net/raw?restype=container",
    "method": "GET",
    "headers": {
      "_headersMap": {
        "x-ms-version": {
          "name": "x-ms-version",
          "value": "2019-12-12"
        },
        "user-agent": {
          "name": "user-agent",
          "value": "MicrosoftAzure Storage Explorer, 1.15.1, win32,  azsdk-js-storageblob/12.2.0-preview.1 (NODE-VERSION v12.13.0; Windows_NT 10.0.19042)"
        },
        "x-ms-client-request-id": {
          "name": "x-ms-client-request-id",
          "value": "9efca422-e865-440c-bf8b-6a1244430852"
        },
        "authorization": {
          "name": "authorization",
          "value": [REDACTED]
        },
        "cookie": {
          "name": "Cookie",
          "value": ""
        }
      }
    },
    "withCredentials": false,
    "timeout": 0,
    "keepAlive": true,
    "decompressResponse": false,
    "requestId": "9efca422-e865-440c-bf8b-6a1244430852",
    "operationSpec": [REDACTED FOR BREVITY]
  }
}
```

**Comment by craxal** [comment]
@saurk Before we can look further into this issue, can you please provide:
- The Storage Explorer version you are using
- Your OS version
- Steps you took to encounter this issue
- How you are connecting to your resource (sign in, attach via Azure AD, attach via SAS, etc.)

**Comment by MRayermannMSFT** [comment]
It looks like @saurk is on 1.15. Please upgrade to the latest storage explorer. Are you behind a proxy? Have you tried this on a different network?

**Comment by MRayermannMSFT** [comment]
Closing due to lack of response. Please open a new issue if you are still having problems.
